sdk/forms/src/main/java/com/klaviyo/forms/bridge/BridgeMessage.kt

```diff
@@ -153,7 +168,7 @@ internal sealed class BridgeMessage {
          * Parse out the android platform deep link
          */
         private fun JSONObject.getDeepLink(): String {
-            val routeString = getString("android")
+            val routeString = optString("android")
```

Comment:
Noticed while updating this, that getString throws, so the intended exception type below is never thrown